docs/install.rst

```diff
@@ -109,48 +109,101 @@ If you make a change to ``media/static-bundles.json``, you'll need to restart Do
 Local Installation
 ------------------
 
-These instructions assume you have Python 3.6+, pip, and NodeJS installed. If you don't have `pip` installed
+These instructions assume you have Python, pip, and NodeJS installed. If you don't have `pip` installed
 (you probably do) you can install it with the instructions in `the pip docs <https://pip.pypa.io/en/stable/installing/>`_.
 
-You need to create a virtual environment for Python libraries:
+Bedrock currently uses Python 3.9.10. The recommended way to install and use that version is
+with `pyenv <https://github.com/pyenv/pyenv>`_ and to create a virtualenv using
+`pyenv-virtualenv <https://github.com/pyenv/pyenv-virtualenv>`_ that will isolate Bedrock's
+dependencies from other things installed on the system.
 
-1. Create a virtual env in the folder `venv` ::
+The following assumes you are on MacOS, using ``zsh`` as your shell and `Homebrew <https://brew.sh/>`_
+as your package manager. If you are not, there are installation instructions for a variety of
+platforms and shells in the READMEs for the two pyenv projects.
 
-    $ python3 -m venv venv
+**Install Python 3.9.10 with pyenv**
 
-2. Activate the virtual env. On Windows, run: venv\Scripts\activate.bat ::
+1. Install ``pyenv`` itself ::
 
-    $ source venv/bin/activate
+    $ brew install pyenv
+
+2. Configure your shell to init ``pyenv`` on start - this is noted in the project's `own docs <https://github.com/pyenv/pyenv>`_, in more detail ::
+
+    $ echo 'eval "$(pyenv init --path)"' >> ~/.zprofile
+
+    % echo 'eval "$(pyenv init -)"' >> ~/.zshrc
+
+3. Restart your login session for the changes to profile files to take effect - if you're not
+using ``zsh``, the ``pyenv`` docs have other routes ::
+
+    $ zsh -l
+
+4. Install Python 3.9.10, then test it's there::
```

Comment:
What's the plan for keeping this document up to date as new Python 3.9.x versions come out? Should it be written more generically as `$PYTHON_VERSION` or say at the time of writing this was 3.9.10?

Reply:
I'll add a note about 3.9.10 and that >=3.9.10 should be OK